### ISSUE TITLE
Final docs cleanup for v1.0.0

### ISSUE THREAD

**Issue by Dunbaratu** [KSP-KOS/KOS#1688]
Just a reminder that whenever there's been lots of doc changes there's almost always a need to go back and fix a few things here and there before release - usually broken links or malformed typesetting markdown.

This is the issue to address that in one final pass before release, only after all the other documentation changes are merged into develop.

**Comment by Dunbaratu** [comment]
This also includes crafting a `changes.rst` section for the release.

**Comment by hvacengi** [comment]
Fixed as part of the release of v1.0.0: https://github.com/KSP-KOS/KOS/tree/v1.0.0
